_sass/_components/blog-post.scss

```diff
@@ -0,0 +1,197 @@
+// Blog post
```

Comment:
Similar comment about `blog_index`. Could we break it into smaller components? Could help to reduce over nesting.